drivers/builtin/include/mbedtls/error_common.h

```diff
@@ -76,17 +65,8 @@ extern void (*mbedtls_test_hook_error_add)(int, int, const char *, int);
  * \param file      file where this error code combination occurred.
  * \param line      line where this error code combination occurred.
  */
-static inline int mbedtls_error_add(int high, int low,
-                                    const char *file, int line)
+static inline int mbedtls_error_add(int high, int low)
```

Comment:
Clang (in the CI) points out we should remove `file` and `line` from the documentation as well.